app/src/main/java/waiter/RouteDirector.java

```diff
@@ -0,0 +1,54 @@
+package waiter;
+
+public class RouteDirector {
+
+    public Route constructGetSimpleGet(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("/simple_get")
+                .method("GET")
+                .status("200 OK")
```

Comment:
Whenever I create a new route to a resource with a `GET` or `HEAD` method will I need to remember to add a "200 OK" status code?

Reply:
As of right now this is the case, the status code and message need to be specified for every route

Reply:
Should we use Enum for this? Usage can be: `StatusCode.OK` or `StatusCode.NotFound` etc.

Reply:
I can look into this, I like the Enum suggestion, thank you! 😄

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/18/commits/c961b693c2652acfc53b02518629f441292d8b50)